### PR TITLE
gradle version upgrades

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -2,14 +2,16 @@
 buildscript {
     repositories {
         jcenter()
+        google()
```
Comment (rogerhu) [inline build.gradle:5]: This should go first

File: build.gradle [hunk 2]
```diff
@@ -2,14 +2,16 @@
...
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.3'
+        classpath 'com.android.tools.build:gradle:3.4.1'
     }
 }
 
 allprojects {
     repositories {
         jcenter()
+        google()
```
Comment (rogerhu) [inline build.gradle:15]: Same here

**Comment by nesquena** [comment]
Thanks

**Comment by rogerhu** [comment]
The build failed.  I think travis.yml needs updating

**Pull request merged**